getting started/README.md

```diff
@@ -0,0 +1,17 @@
+# Getting Started
+
+This document covers the fundamental approach to tackle any problem arriving at the initial phase of the journey.
+
+
+
+## Online resources
+#### Try not to take shortcut
+1. https://www.youtube.com/watch?v=3xq96I9IUXw
+
+
+
+#### Books
```

Comment:
```suggestion
## Books
```

Reply:
Made changes in the latest commit. Thanks